benchmarks/data-caching/client/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM cloudsuite/debian:base-os
```

Comment:
Please rebase and bring the latest changes to your branch and update the travis file to build the new image.